dexcom/fetch/translate.go

```diff
@@ -388,8 +376,26 @@ func translateEGVToDatum(egv *dexcom.EGV, unit *string, rateUnit *string) data.D
 				"threshold": dexcom.EGVValuePinnedMgdLMaximum,
 			}}
 		}
+	case dexcom.EGVUnitMmolL:
+		if *datum.Value < dexcom.EGVValuePinnedMmolLMinimum {
+			datum.Value = pointer.FromFloat64(dexcom.EGVValuePinnedMmolLMinimum - 0.1) //check this
```

Comment:
seeking feedback on this - just went for arbitory minimal value

Reply:
Given that the current API may only return `mg/dL`, this seems like a reasonable low-risk approach.